package.json

```diff
@@ -16,7 +16,7 @@
   "license": "AGPLv3",
   "dependencies": {
     "angularjs-dragula": "2.0.0",
-    "dompurify": "1.0.9",
+    "dompurify": ">=2.2.0",
```

Comment:
Why not using a fixed version?

Reply:
Yes correct @chibenwa . I would like that we use the tilde range to specify that we only accept patch level upper versions. https://www.npmjs.com/package/semver#tilde-ranges-123-12-1